test/gtest/ucp/test_ucp_atomic.cc

```diff
@@ -134,7 +134,7 @@ class test_ucp_atomic : public test_ucp_memheap {
     }
 
     static unsigned default_num_iters() {
-        return ucs_max(100 / ucs::test_time_multiplier(), 1);
+        return (128 * 1024) / ucs::test_time_multiplier();
```

Comment:
for completeness, shall we still keep explicitly making sure it's > 1?

Reply:
In many places where we use `test_time_multiplier` we don't compare result with 1 and I think it doesn't have sense to check it if number of iterations is 128K.

Reply:
i think @tvegas1 suggestion is valid, given that `ucs::test_time_multiplier()` can be changed later in a way that num_iters will be 0 here

Reply:
I think to avoid such case we need to introduce something like `size_t ucs::iters_num(size_t input_num_iters)` which should care about dividing to `ucs::test_time_multiplier()` and compare result with 0 since as I said before there are many places where such error can occur and FMPOV it is better to fix them all together.

Reply:
i do not see how having issuea in other places justify introducing one more, but it's a minor comment, feel free to ignore

Reply:
```suggestion
        return ucs_max((128 * 1024) / ucs::test_time_multiplier(), 1);
```